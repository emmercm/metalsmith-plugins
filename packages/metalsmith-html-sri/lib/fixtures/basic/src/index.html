<!DOCTYPE html>
<html lang="en">
<head>
    <title>Index</title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="./style.css">
    <link rel="stylesheet" href="/style.css">
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.8.2/css/all.css">
</head>
<body>
<script src="script.js"></script>
<script src="./script.js"></script>
<script src="/script.js"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"></script>
</body>
</html>
