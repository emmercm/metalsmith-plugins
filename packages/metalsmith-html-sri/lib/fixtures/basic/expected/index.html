<!DOCTYPE html><html lang="en"><head>
    <title>Index</title>
    <link rel="stylesheet" href="style.css" integrity="sha384-rVWZyZaSdPEvKfhMfMQVa5Dmhq0Nm1Kn+7IxuWfa3oxHpTQJ3OCEBDN6iQLJzAcI">
    <link rel="stylesheet" href="./style.css" integrity="sha384-rVWZyZaSdPEvKfhMfMQVa5Dmhq0Nm1Kn+7IxuWfa3oxHpTQJ3OCEBDN6iQLJzAcI">
    <link rel="stylesheet" href="/style.css" integrity="sha384-rVWZyZaSdPEvKfhMfMQVa5Dmhq0Nm1Kn+7IxuWfa3oxHpTQJ3OCEBDN6iQLJzAcI">
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
    <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.8.2/css/all.css">
</head>
<body>
<script src="script.js" integrity="sha384-OWEZQZeSlY/MFePsScouReLsIME1kn4bId5gGFXmZrK4gbfN7N3k0RVGPH3zxlIb"></script>
<script src="./script.js" integrity="sha384-OWEZQZeSlY/MFePsScouReLsIME1kn4bId5gGFXmZrK4gbfN7N3k0RVGPH3zxlIb"></script>
<script src="/script.js" integrity="sha384-OWEZQZeSlY/MFePsScouReLsIME1kn4bId5gGFXmZrK4gbfN7N3k0RVGPH3zxlIb"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js" integrity="sha384-xrRywqdh3PHs8keKZN+8zzc5TX0GRTLCcmivcbNJWm2rs5C8PRhcEn3czEjhAO9o" crossorigin="anonymous"></script>


</body></html>